src/main/java/fr/pickaria/pterodactylpoweraction/configuration/YamlConfiguration.java

```diff
@@ -70,6 +70,11 @@ public Optional<String> getPterodactylServerIdentifier(String serverName) {
             Object configuration = getServerConfiguration(serverName);
             if (configuration instanceof String) {
                 return Optional.of((String) configuration);
+            } else if (configuration instanceof Map serverConfiguration) {
+                Object identifier = serverConfiguration.get("identifier");
+                if (identifier instanceof String) {
+                    return Optional.of((String) identifier);
+                }
```

Comment:
From the looks of it, there are now two ways to declare a server in the configuration file. While this is good for maintaining backwards compatibility. You must ensure that this is also documented in the docs.